changelogs/fragments/314-postgresql_info-quote-show-args.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- postgresql_info - make arguments passed to SHOW command properly quoted to prevent the interpreter evaluating them for reserved keywords (https://github.com/ansible-collections/community.postgresql/issues/314).
```

Comment:
Wasn't just issue with reserved words. I believe we ran into this because of special characters since almost all third-party extensions have periods in their variable names to avoid collisions.